python/cudf/cudf/core/frame.py

```diff
@@ -298,6 +293,12 @@ def empty(self):
         Series([], dtype: float64)
         >>> s.empty
         True
+
```

Comment:
this part still need improvement.

Reply:
Because many method in frame are used in Both DataFrame and Series the documentation will likely generate both for DataFrame and Series with the same class name that can lead to confusion.

Reply:
Because I'm lack understanding in `delegateMixin` `Operation` class so i resolve it like this. i will help change it later thank you.

Reply:
The alternative is add simple code such as `class.method.__doc__ = formatDocstring`